### PR TITLE
JMAP forwards documentation

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: docs/modules/ROOT/pages/tmail-backend/features/index.adoc [hunk 1]
```diff
@@ -59,4 +59,17 @@ Contacts can be stored in many ways in TMail:
 
 The user is able to autocomplete over his personal contacts and domain contacts via a JMAP method.
 
-xref:tmail-backend/features/contactAutocomplete.adoc[Read more]...
\ No newline at end of file
+xref:tmail-backend/features/contactAutocomplete.adoc[Read more]...
+
+== Forwards
+
+Tmail allows users to configure their account to forward mails to other mail addresses via JMAP methods.
```
Comment (quantranhong1999) [inline docs/modules/ROOT/pages/tmail-backend/features/index.adoc:66]:
```suggestion
TMail allows users to configure their account to forward mails to other mail addresses via JMAP methods.
```

File: docs/modules/ROOT/pages/tmail-backend/features/index.adoc [hunk 2]
```diff
@@ -59,4 +59,17 @@ Contacts can be stored in many ways in TMail:
...
+
+This simple feature can be useful when one of your users is on vacation, or leaves your organization if it's easier
+to just forward emails in some cases to other people that can treat them.
```
Comment (quantranhong1999) [inline docs/modules/ROOT/pages/tmail-backend/features/index.adoc:69]:
```suggestion
This simple feature can be useful when one of your users is on vacation or leaves your organization then it's easier
to just forward emails in some cases to other people that can treat them.
```
?

**Review by quantranhong1999 — COMMENTED** [review]
Look good to me.

File: docs/modules/ROOT/pages/tmail-backend/features/index.adoc [hunk 3]
```diff
@@ -59,4 +59,17 @@ Contacts can be stored in many ways in TMail:
 
 The user is able to autocomplete over his personal contacts and domain contacts via a JMAP method.
 
-xref:tmail-backend/features/contactAutocomplete.adoc[Read more]...
\ No newline at end of file
+xref:tmail-backend/features/contactAutocomplete.adoc[Read more]...
+
+== Forwards
```
[the remark this answers is not in the document]
Comment (Arsnael) [reply docs/modules/ROOT/pages/tmail-backend/features/index.adoc:64]: An icon? You really mean an icon, or a drawing showing how the forward works?

File: docs/modules/ROOT/pages/tmail-backend/jmap-extensions/forwards.adoc [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+}
+....
+
+== Methods
+
+=== Forward/get
+
+This is a standard “/get” method for the Forward object:
+
+- The `state` property reflects the internal event sourcing state.
```
[the remark this answers is not in the document]
Comment (Arsnael) [reply docs/modules/ROOT/pages/tmail-backend/jmap-extensions/forwards.adoc:44]: Well we send a state back in the method... after yeah maybe it's a dummy state/not used. I need to double check.

File: docs/modules/ROOT/pages/tmail-backend/features/index.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (chibenwa) [reply docs/modules/ROOT/pages/tmail-backend/features/index.adoc:64]: I mean pick the one you love in https://www.google.com/search?q=forward&sxsrf=ALiCzsYwQgkEYm5UjnMg_Ae8UYHoVJaUNg:1652788186935&source=lnms&tbm=isch&sa=X&ved=2ahUKEwi-spXmu-b3AhUOdZQKHeRFBBMQ_AUoAXoECAEQAw&biw=1858&bih=968&dpr=1

File: docs/modules/ROOT/pages/tmail-backend/jmap-extensions/forwards.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (chibenwa) [reply docs/modules/ROOT/pages/tmail-backend/jmap-extensions/forwards.adoc:44]: Maybe just remove that line.

**Review by chibenwa — APPROVED** [review]

**Pull request merged**